### ISSUE TITLE
Estimate of time to complete or progress bar?

### ISSUE THREAD

**Issue by ben-rabid** [GOVTNZ/web-standards#7]
This is great work Katherine & others - well done.

As I was scanning over it I kinda got the feeling testing 2-3 sites with 20 pages to someone unfamiliar could take a few hours, but I couldn't find something that might indicate a rough time so testers could plan accordingly.

As an enhancement in the same direction - perhaps the presentation of the assessment could be in pages with Next/Previous and a loading bar at the top, so testers can get a feel for how far through the process they are? Certainly not something that's needed but it came to mind.

**Comment by kbarcham** [comment]
Hi Ben.
Thanks for the feedback and for taking the time to look over the document.

We originally wondered about putting something like “Time to complete” for each test but that felt a little too granular.
We’ve talked about some better options and will add a section to the overview about how long it might take to test a page. And a note that as you get more experienced as you go through pages, your time to complete will probably decrease. Do you think that would help someone plan their work accordingly?

Also do you have any feeling about how long it might take someone who hasn’t used the tools before, to test a page?

**Comment by kbarcham** [comment]
Closed in error.

**Comment by ben-rabid** [comment]
@kbarcham I agree, you don't want to get too granular on the timings. It's hard for me to take a guess as the time will vary each time, which is why the idea of a progress bar appeals - it will progress at the rate the user moves. You could allocate more width to the tasks that take longer.

It would not have to be live, even static images would be enough as a user scrolls.

**Comment by jasonkiss** [comment]
Thanks for the suggestions, @ben-rabid. Would you suggest having each section and each test on its own page? It'll add more navigation, but might make things clearer. And we can add some kind of progress indicator. Do you see that progress indicator only for the tests, or for all sections of the methodology?

It's difficult to say how one will perform the assessment: will they go through all tests for each standard for each page, or will they perform the first test across all 20 pages, then move on to the next test, etc.? Will they do all the Accessibility Standard tests, and then go back and run all the Usability Standard tests? 

As such, if we do add some kind of progress indicator, all it would indicate, I think, is what section of the methodology or test you are on within the methodology or collection of tests as a whole. That might still be useful, though.

**Comment by ben-rabid** [comment]
@jasonkiss good questions. I can't say with any certainty how people will do the assessment, but I don't think it matters for a progress bar. The user can either see it moving slowly as they do the same test 20 times or quickly but have to do it multiple times.
